src/objectivec/oga_internal.h

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "cxx_api.h"
+#import "error_utils.h"
+#import "ort_genai_objc.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OGAModel ()
+
+- (const OgaModel&)CXXAPIOgaModel;
+
+@end
+
+@interface OGATokenizer ()
+
+- (const OgaTokenizer&)CXXAPIOgaTokenizer;
+
+@end
+
+@interface OGASequences ()
+
+- (nullable instancetype)initWithError:(NSError**)error;
+- (instancetype)initWithCXXPointer:(std::unique_ptr<OgaSequences>)ptr;
```

Comment:
should these also have `NS_DESIGNATED_INITIALIZER`?

Reply:
sure why not.